ocdaction/challenges/views.py

```diff
@@ -27,23 +27,19 @@ def challenge_list(request):
         '-updated_at'
     )[:10]
 
-    for challenge in challenges_not_in_progress:
-        print(challenge.in_progress)
-
     challenges_in_progress_sorted = sorted(
         challenges_in_progress.all(),
         reverse=True,
         key=lambda c: int(c.get_latest_initial_anxiety_level())
     )
 
```

Comment:
This should be sorting the challenges not in progress:
`   challenges_not_in_progress_sorted = sorted(
        challenges_not_in_progress.all(),
        reverse=True,
        key=lambda c: int(c.get_latest_initial_anxiety_level())
    )`